akka-actor-testkit-typed/src/test/scala/akka/actor/testkit/typed/scaladsl/ManualTimerExampleSpec.scala

```diff
@@ -83,21 +85,22 @@ class ManualTimerExampleSpec extends AbstractActorSpec {
             case SlowThenBump(nextCount) ⇒
               manualTime.timePasses(interval)
               timer.startPeriodicTimer("T", Tick(nextCount), interval)
+              probe.ref ! SlowThenBumpAck
               Behaviors.same
           }
         }
       }
 
       val ref = spawn(behavior)
-
       manualTime.timePasses(11.millis)
       probe.expectMessage(Tock(1))
 
-      // next Tock(1) enqueued in mailboxed, but should be discarded because of new timer
+      // next Tock(1) enqueued in mailbox, but should be discarded because of new timer
       ref ! SlowThenBump(2)
-      manualTime.expectNoMessageFor(interval, probe)
+      probe.expectMessage(SlowThenBumpAck)
```

Comment:
Race condition was between the next line (which advances the timer) and the timer advancement in the actor.